test_murmur_hash fails on s390x architecture

While running the tests on the s390x architecture for a Fedora build, I'm seeing this failure.

```
======================================================================
FAIL: test_murmur_hash (tests.sqlite.TestMurmurHash.test_murmur_hash)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/peewee-3.15.2/tests/sqlite.py", line 1576, in test_murmur_hash
    self.assertHash('testkey', 2871421366)
  File "/builddir/build/BUILD/peewee-3.15.2/tests/sqlite.py", line 1573, in assertHash
    self.assertEqual(cursor.fetchone()[0], e)
AssertionError: 3876111420 != 2871421366
----------------------------------------------------------------------
```

This passes on all the other architectures that Fedora builds for (i686, x86_64, aarch64, ppc64le).  I was able to reproduce this error in the following environments:

- [Fedora 38](https://koji.fedoraproject.org/koji/taskinfo?taskID=92255140), Python 3.11.0~rc2, sqlite 3.39.3
- [Fedora 37](https://koji.fedoraproject.org/koji/taskinfo?taskID=92254905), Python 3.11.0~rc2, sqlite 3.39.2
- [Fedora 36](https://koji.fedoraproject.org/koji/taskinfo?taskID=92254910), Python 3.10.6, sqlite 3.36.0
- [Fedora 35](https://koji.fedoraproject.org/koji/taskinfo?taskID=92254914), Python 3.10.6, sqlite 3.36.0
- [RHEL 9](https://koji.fedoraproject.org/koji/taskinfo?taskID=92254798), Python 3.9.10, sqlite 3.34.1

--- a/tests/sqlite.py
+++ b/tests/sqlite.py
@@ -1572,6 +1572,7 @@ class TestMurmurHash(ModelTestCase):
         cursor = self.database.execute(query)
         self.assertEqual(cursor.fetchone()[0], e)
 
+    @skip_if(sys.byteorder == 'big', 'fails on big endian')
     def test_murmur_hash(self):
         self.assertHash('testkey', 2871421366)
         self.assertHash('murmur', 3883399899)

